soc/realtek/ec/rts5912/reg/reg_wdt.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_WDT_H
+#define ZEPHYR_SOC_REALTEK_RTS5912_REG_WDT_H
+
+/**
+ * @brief WDT Controller (WDT)
+ */
+
+typedef struct {
+	volatile uint32_t CTRL;
+	const volatile uint32_t STS;
+	volatile uint32_t CNT;
+	volatile uint32_t DIV;
+	volatile uint32_t INTEN;
+} WDT_Type;
+
+/* CTRL */
+#define WDT_CTRL_EN_Pos         (0UL)
+#define WDT_CTRL_EN_Msk         (0x1UL)
```

Comment:
Because these are all single-bit values, no need to append the `_Msk` suffix.  You should also use the BIT() macro here.
```suggestion
#define WDT_CTRL_EN        BIT(0)
```

Reply:
Fixed